CHANGELOG.md

```diff
@@ -20,6 +20,10 @@ To install release candidates run `[sudo] gem install cocoapods --pre`
 
 ##### Bug Fixes
 
+* Umbrella header import path gets wrong when `header_dir` is specified in PodSpec + try to build statically + Modular header is enabled
```

Comment:
nit: 2 empty spaces at the end of this line

Reply:
OK, I'll add it with squash!

Reply:
Done!